torch/_torch_docs.py

```diff
@@ -6999,10 +6999,10 @@ def merge_dicts(*dicts):
 """.format(**factory_like_common_args))
 
 add_docstr(torch.randperm,
-           r"""
-randperm(n, \*, generator=None, out=None, dtype=torch.int64, layout=torch.strided, device=None, requires_grad=False,
-    pin_memory=False) -> LongTensor
-
+           """
+randperm(n, *, generator=None, out=None, dtype=torch.int64,layout=torch.strided, \
+device=None, requires_grad=False, pin_memory=False) -> LongTensor
```

Comment:
Let's fix this while we're here and replace "LongTensor" with "Tensor." "LongTensor" is an antiquated term and it's not even correct in this instance because this function supports the `dtype` kwarg.